lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -20,6 +20,9 @@ def self.provision_execute(_params = {}, _secrets = {}, context = {})
           return BuiltinRunnner.error!({}, :cause => "Missing MiqRequestTask id")   if object_id.nil?
 
           miq_request_task = ::MiqRequestTask.find_by(:id => object_id.to_i)
+          new_options = miq_request_task.options.merge(context.input.symbolize_keys)
+          miq_request_task.update(:options => new_options)
```

Comment:
```suggestion
          miq_request_task.update!(:options => new_options)
```

Reply:
Also these lines should come _after_ the 2 raise's below which check for the presence of the miq_request_task and the class type.